JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -647,7 +632,7 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
 
 	
 	if (!is.null(shade)) {
-	  datFilter <- datDens[datDens[["x"]] >= shade[1] & datDens[["x"]] <= shade[2], ]
+	  datFilter <- datDens[datDens[["x"]] >= shade[1] && datDens[["x"]] <= shade[2], ]
```

Comment:
I think here you probably want a single `&`

`&&`:
- expects a single boolean on either side => `T && F`
- returns a single boolean
- stops evaluation of the statement if the left hand side returns `FALSE` => `if (FALSE && stop("we reached the stop")) {}`
- is useful in if statements

`&`:
- is vectorized and can work with multiple boolean values on either side => `c(T, T, F) & c(F, F, T)`
- returns a vector of booleans
- always evaluates both sides => `if (FALSE & stop("we reached the stop")) {}`
- is useful in indexing to select multiple elements

Reply:
thanks for the clarification :)